src/Utils.jl

```diff
@@ -412,4 +412,11 @@ function sort_by_creation_time(files)
     return sort(files, by = x -> stat(x).ctime)
 end
 
+"""
+    unwrap(x::Val{N})
+
+Unwrap the value in `Val{N}` to get the value at runtime.
```

Comment:
```suggestion
Unwrap the value in `Val{N}` to get `N`.
```
It's a compile-time parameter, not really a runtime parameter